pkg/modules/application_mobility_test.go

```diff
@@ -516,6 +516,35 @@ func TestAppMobilityVelero(t *testing.T) {
 
 			return true, false, tmpCR, sourceClient, operatorConfig
 		},
+
+		"success - upgrading with old node agent with valid old version": func(*testing.T) (bool, bool, csmv1.ContainerStorageModule, ctrlClient.Client, utils.OperatorConfig) {
+			customResource, err := getCustomResource("./testdata/cr_application_mobility.yaml")
+			if err != nil {
+				panic(err)
```

Comment:
Is it normal to panic with these tests?

Reply:
It is consistent amongst all the unit tests here. Leaving as is.